src/Components/Reports/Common/SystemsTablePage.js

```diff
@@ -6,7 +6,7 @@ import styles from './styles';
 
 const columnBuilder = ({ value, style, key }) => <Text key={key} style={style}>{value}</Text>;
 
-const headerContent = ['Name', 'OS', 'CPU utilization', 'Memory utilization', 'I/O utilization', 'Suggestions', 'State', 'Last reported'];
+const headerContent = ['Name', 'Group', 'OS', 'CPU utilization', 'Memory utilization', 'I/O utilization', 'Suggestions', 'State', 'Last reported'];
```

Comment:
Could you include group tests -  [SystemsFirstPage.test.js](https://github.com/RedHatInsights/ros-frontend/blob/main/src/Components/Reports/Common/SystemsFirstPage.test.js)

Reply:
@r14chandra have pushed the changes.